packages/edit-site/src/components/global-styles/font-library-modal/google-fonts-confirm-dialog.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	Card,
+	CardBody,
+	__experimentalText as Text,
+	__experimentalSpacer as Spacer,
+} from '@wordpress/components';
+
+function GoogleFontsConfirmDialog() {
+	const handleConfirm = () => {
+		// eslint-disable-next-line no-undef
+		localStorage.setItem(
+			'wp-font-library-google-fonts-permission',
+			'true'
+		);
+		window.dispatchEvent( new Event( 'storage' ) );
+	};
+
+	return (
+		<div className="font-library__google-fonts-confirm">
+			<Card>
+				<CardBody>
+					<Text as="h3">Connect to Google Fonts</Text>
+					<Spacer margin={ 6 } />
+					<Text as="p">
+						{ __(
+							'To install Google Fonts you have to give permission to connect directly to Google Servers. The fonts install will be downloaded from Google servers at install time and stored in your server.'
+						) }
+					</Text>
+					<Spacer margin={ 3 } />
+					<Text as="p">
+						{ __(
+							'If you prefer to avoid this, you can upload font files.'
```

Comment:
```suggestion
							'You can alternatively upload files directly on the Library tab.'
```
Possible refinement.